packages/gatsby-source-filesystem/src/utils.js

```diff
@@ -58,3 +59,32 @@ export function slash(path) {
 
   return path.replace(/\\/g, `/`)
 }
+
+/**
+ * createFilePath
+ * --
+ *
+ * @param  {String} directory
+ * @param  {String} filename
+ * @param  {String} url
+ * @return {String}
+ */
+export function createFilePath(directory, filename, ext) {
+  return path.join(directory, `${filename}${ext}`)
+}
+
+/**
+ * md5Buffer
+ * --
+ * Hashes the contents of a buffer using MD5
+ *
+ *
+ * @param {Buffer}            buffer
+ * @return {String}           hash
+ */
+export function md5Buffer(buffer) {
```

Comment:
Would you mind switching to our content-digest-helper located in gatsby package?
`const { createContentDigest } = require(`gatsby/utils`)`. It's basically the same thing what you wrote but to keep it consistent we would love that you use that one.

Thanks for understanding